Sources/AWSSDKSwiftCore/HTTP/AsyncHTTPClient.swift

```diff
@@ -18,19 +18,71 @@ import NIO
 
 /// comply with AWSHTTPClient protocol
 extension AsyncHTTPClient.HTTPClient: AWSHTTPClient {
+
+    /// write stream to StreamWriter
+    private func writeToStreamWriter(
+        writer: HTTPClient.Body.StreamWriter,
+        size: Int?,
+        on eventLoop: EventLoop,
+        stream: @escaping (EventLoop)->EventLoopFuture<ByteBuffer>) -> EventLoopFuture<Void> {
```

Comment:
this is not really a stream. it is more a get more data callback. do we maybe want to name it. `getMoreData`.

Reply:
`getData` should be enough, `getMoreData` implies we've already had some

Reply:
done